neurodamus/connection_manager.py

```diff
@@ -725,15 +725,25 @@ def target_gids(gids):
 
     def _get_conn_stats(self, _src_target, dst_target):
         """Estimates the number of synapses per type for the given destination target
+
+        Args:
+            _src_target: not used
+            dst_target: The target to estimate synapses for
```

Comment:
At this point we know that `_src_target` wont likely ever be used. I would drop it from the signature and modify the comment
```suggestion
    def _get_conn_stats(self, dst_target):
        """Estimates the number of synapses per type for the given destination target.
        Note: measurement is done without any restriction on the source target. As so
        it can avoid counting the same synapse twice by keeping track of target cells alone.

        Args:
            dst_target: The target to estimate synapses for
```